### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by jssol**
### Project requirements:

- [x] Setup your project with npm
- [x] Re-create your Awesome books app following the rules
- [x] Display the current date below the navigation bar. - For this requirement, you should use a library - [Luxon](https://moment.github.io/luxon/#/install)

### Useful links:
- [Project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md)
- [Live Site URL](https://jssol.github.io/awesome-books-es6/)

### CODE REVIEW

File: modules/content.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        title: 'Coding',
+      },
+      {
+        author: 'Lumumba',
+        title: 'Freedom',
+      },
+    ];
+  }
+
+  setLocalStore() {
```
Comment (TedLivist) [inline modules/content.js]: - [x] Kindly refactor this function to use ES6 syntax (arrow function).

File: modules/content.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    localStorage.setItem('bookStore', JSON.stringify(this.books));
+  }
+
+  removeBook(index) {
```
Comment (TedLivist) [inline modules/content.js]: - [x] Kindly refactor to arrow function

File: modules/content.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    this.books.splice(index, 1);
+    this.setLocalStore(this.books);
+  }
+
+  addBook(title, author) {
```
Comment (TedLivist) [inline modules/content.js]: - [x] Kindly refactor to arrow function

File: modules/content.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    const book = {
+      author,
+      title,
+    };
+    this.books.push(book);
+    this.displayBooks();
+    this.setLocalStore(this.books);
+  }
+
+  displayBooks() {
```
Comment (TedLivist) [inline modules/content.js]: - [x] Kindly refactor to arrow function

File: README.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+### Usage
+
+This project helps you store your favorite books right in your navigator.
+
+### Run tests
+
+For:
+
+- Eslint: run `npx eslint .`
+- Eslint: run `npx hint .`
```
Comment (TedLivist) [inline README.md]: - [x] I think the correct linter being run here is Webhint. I'd recommend you fix the typo error.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+## Live Demo
+
+[Live Demo Link](https://jssol.github.io/awesome-books-es6/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
```
Comment (TedLivist) [inline README.md]: - [x] Let's clean up the documentation. Since this **Prerequisites** section has no description, kindly remove it or add a description.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @jssol,

Good job so far!
✔️ Nice UI
✔️ Use of import/export
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elyor-doniyorov — APPROVED** [review]
Hi @jssol, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS APPROVED ✅ 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**